Tasks/ANT/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -39,7 +39,7 @@
   "loc.input.label.jdkArchitecture": "JDK Architecture",
   "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of the JDK.",
   "loc.messages.LocateJVMBasedOnVersionAndArch": "Locate JAVA_HOME for Java %s %s",
-  "loc.messages.FailedToLocateSpecifiedJVM": "Failed to find specified JDK version. Please make sure environment variable '%s' exists and is set to the location of a corresponding JDK.",
```

Comment:
I think the old error text may be useful for people running their own agents. If I install Java on a Mac or Linux machine will JAVA_HOME_9_X64 be set for me? I figure this is an extra manual step we require, unless you're on a Windows machine.

Reply:
Good point! I updated the comment to still include the original error.